aws/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -1,6 +1,6 @@
 # Self-hosted Kubernetes assets (kubeconfig, manifests)
 module "bootkube" {
-  source = "github.com/kinvolk/terraform-render-bootkube?ref=30b2abcb1266ce9cedc92ddb3c6d8a4bbcbe7da0"
+  source = "github.com/kinvolk/terraform-render-bootkube?ref=06bc0786082cc5203ff8867246b1bcd6c1612d83"
```

Comment:
Shouldn't we point here to the merge commit (which does not exist yet)?

Reply:
But then how would I test it? This is here so that I can test on the CI and once terraform-render-bootkube is merged, it will be updated again to point to the merge commit.

Reply:
Fixed it :+1: thanks